fastly/resource_fastly_service_dynamic_snippet_content_v1_test.go

```diff
@@ -153,44 +160,6 @@ func TestAccFastlyServiceDynamicSnippetContentV1_normal_snippet_is_not_removed(t
 	})
 }
 
-func TestAccFastlyServiceDynamicSnippetContentV1_import(t *testing.T) {
```

Comment:
I presume this is removed as it's replaced by TestCheckTypeSetElemNestedAttrs

Reply:
From the linter it is suggested that Import state tests be implemented as a step within other acceptance tests.  This way the test can confirm that configuration can be imported.  In our case the Import state tests we a copy of basic and so these have been combined.  https://github.com/bflad/tfproviderlint/tree/main/passes/AT002.  This will remove the need to provision another service just to check that it can be imported.